src/replicache.ts

```diff
@@ -808,6 +813,38 @@ export class Replicache<MD extends MutatorDefs = {}> {
     this._pullConnectionLoop.send();
   }
 
+  /**
+   * Applies an update from the server to Replicache.
+   * Throws an error if cookie does not match. Host should re-init.
```

Comment:
> Host should re-init.

Can you expand on this?

Reply:
I think this should say *client* should re-init: the server has sent us an update on top of a cookie it thinks we have but we don't. The client disconnecting and re-registering ensures that the server knows the latest cookie the client has, because it is included in the registration request.

Reply:
This comment is saying that the current semantics are that if this method throws, caller (the app) should re-connect to its server because the cookies have gotten out of sync.

This is temporary -- in the near future I imagine Replicache implementing poke as a first-class concept and in that case, there would be a new `connect()` callback analagous to the current `pusher()` and `puller()` callbacks that Replicache would call to get host to re-connect.